rfcs/0024-ledger-private-key.md

```diff
@@ -0,0 +1,132 @@
+## Summary
+[summary]: #summary
+
+Ledger devices don't maintain any state/storage between use, and only give
```

Comment:
Maybe make clear this is a particular hardware device, and not just a ledger in the generic sense